### PR TITLE
Added create site command, removed 4.0.0 version

### PR DESCRIPTION
**Pull request by liorblob**
scans will be possible in 3.6 but without polling, will return immediately

### CODE REVIEW

File: Integrations/integration-Rapid7_Nexpose.yml [hunk 1]
```diff
@@ -1445,6 +1499,31 @@ script:
...
+      description: <string> The site name. Name must be unique.
+    - name: description
+      description: <string> The site's description.
+    - name: assets
+      required: true
+      description: Multiple criteria of <string> Specify asset addresses to be included
+        in site scans
+      isArray: true
+    - name: scanTemplateId
+      description: <string> The identifier of a scan template. Default scan template
```
Comment (anara123) [inline Integrations/integration-Rapid7_Nexpose.yml]: where I can get this scanTemplateId, any specific command should I run?

File: Integrations/integration-Rapid7_Nexpose.yml [hunk 2]
```diff
@@ -1445,6 +1499,31 @@ script:
       description: Common Vulnerabilities and Exposures ids
     description: Returns the details and possible remediations for an asset's given
       vulnerability.
+  - name: nexpose-create-site
```
Comment (anara123) [inline Integrations/integration-Rapid7_Nexpose.yml]: I don't see any outputs?

**Comment by anara123** [comment]
@liorblob also update the docs (open an issue)

File: Integrations/integration-Rapid7_Nexpose.yml (hunk 1 above)
Comment (liorblob) [reply Integrations/integration-Rapid7_Nexpose.yml]: yes, `nexpose-get-report-templates`, I will add to the description

File: Integrations/integration-Rapid7_Nexpose.yml (hunk 2 above)
Comment (liorblob) [reply Integrations/integration-Rapid7_Nexpose.yml]: Will add

**Comment by liorblob** [comment]
I'm updating the doc to the new template

**Pull request merged**